common/src/main/java/io/netty/util/ResourceLeakDetector.java

```diff
@@ -119,7 +121,7 @@ public static void setEnabled(boolean enabled) {
      * Returns {@code true} if resource leak detection is enabled.
      */
     public static boolean isEnabled() {
-        return getLevel().ordinal() > Level.DISABLED.ordinal();
```

Comment:
@Scottmitch I don't understand what you are trying to say here. Both level and ENABLED are final here already...